plugins/rabbitmq.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: rabbitmq
```

Comment:
This name makes sense and seems justified given your membership in the project. :+1: